tasks/github_tasks.py

```diff
@@ -260,6 +261,15 @@ def assign_team_label(_, pr_id=-1):
         print('No team found')
         return
 
+    _assign_pr_team_labels(gh, pr_id, teams)
+
+
+def _assign_pr_team_labels(gh, pr_id, teams):
+    """
+    Assign team labels (team/team-name) for each team (@dataog/team-name)
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
    Assign team labels (team/team-name) for each team (@datadog/team-name)
```